src/wasm-lib/kcl/src/execution/mod.rs

```diff
@@ -854,6 +862,7 @@ impl Plane {
                 y_axis: *y_axis,
                 z_axis: *z_axis,
                 value: PlaneType::Custom,
+                units: exec_state.length_unit(),
```

Comment:
Hmm, you can't specify the units on the coordinates themselves, huh? Is the next step in the units plan to allow this? Unclear to me how this will all work together, but I trust you that you'll figure it out.

Reply:
Yeah, one of the next steps is to add units to floats and coords etc

Reply:
It's a really huge, messy PR so I'm picking off any bits that are relatively self-contained like this